### PR TITLE
feat: integrate with okta to verify the token

### PR DESCRIPTION
**Pull request by ritikrajdev**
Integrate okta with our server to verify the token.
Our Swagger doc is also showing the type of authentication required.

image

### CODE REVIEW

**Review by Abhishek-Kharpal — COMMENTED** [review]
LGTM

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DATABASE_URL='postgres://postgres:postgres@localhost:5432/postgres'
+OKTA_CLIENT_ID='0oa1x9x9x9x9x9x9x9x9'
+# do not start this with https://
+OKTA_DOMAIN='dev-1248769.okta.com'
```
Comment (sugumar-mani) [inline .env.example:4]: dummify this value `dev-1248769.okta.com` like `<teamid>.okta.com`

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DATABASE_URL='postgres://postgres:postgres@localhost:5432/postgres'
+OKTA_CLIENT_ID='0oa1x9x9x9x9x9x9x9x9'
```
Comment (sugumar-mani) [inline .env.example:2]: make it `<okta-client-id-value>`

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,4 @@
+DATABASE_URL='postgres://postgres:postgres@localhost:5432/postgres'
```
Comment (sugumar-mani) [inline .env.example:1]: postgres://<user>:<pass>@<host>:<port>/<db-name>

**Review by sugumar-mani — APPROVED** [review]
Please check my comments before merge

File: .env.example (hunk 1 above)
Comment (ritikrajdev) [reply .env.example]: Hi sugumar, this has been resolved in [`07e4f69` (#33)](https://github.com/tech-university-india/abd-backend/pull/33/commits/07e4f69e1bc37362843b2ba37f373307ac43adb6)

File: .env.example (hunk 2 above)
Comment (ritikrajdev) [reply .env.example]: This has been resolved in [`07e4f69` (#33)](https://github.com/tech-university-india/abd-backend/pull/33/commits/07e4f69e1bc37362843b2ba37f373307ac43adb6)

File: .env.example (hunk 3 above)
Comment (ritikrajdev) [reply .env.example]: This has been changed in [`07e4f69` (#33)](https://github.com/tech-university-india/abd-backend/pull/33/commits/07e4f69e1bc37362843b2ba37f373307ac43adb6)

**Pull request merged**